ansible_base/activitystream/models/entry.py

```diff
@@ -0,0 +1,83 @@
+from functools import partial
+
+from django.contrib.contenttypes.fields import GenericForeignKey
+from django.contrib.contenttypes.models import ContentType
+from django.db import models
+from django.db.models.signals import m2m_changed, post_save, pre_delete, pre_save
+from django.utils.translation import gettext_lazy as _
+
+from ansible_base.activitystream.signals import activitystream_create, activitystream_delete, activitystream_update, activitystream_m2m_changed
+from ansible_base.lib.abstract_models import CommonModel
+
+
+class Entry(CommonModel):
```

Comment:
If this inherits from `CommonModel` that adds a lot of fields that might not make sense here.

For the RBAC assignments, I wanted assignments to be immutable so I used the common model and set `modified_by = None` and `modified_on = None`. That's worked well, I like it.

Even so, I feel like this stretches the definition of `created_by` a little bit? Does that then become the attributable user for this entry? I saw you ordered by `created_on`, so it leads me to think that. That would still be okay if intended and documented, but I'd also expect immutability.

Reply:
I feel like using `CommonModel` is the right idea here, it gets us some stuff for almost-free (the system user logic, etc.).

I agree that I don't expect activity stream entries to ever be modified, so I'm fine forcing immutability in `save()`, and nullifying the `modified_*` fields.